taipy/config/common/_template_handler.py

```diff
@@ -51,11 +51,11 @@ def _replace_template(cls, template, type, required, default):
                 if required:
                     raise MissingEnvVariableError(f"Environment variable {var} is not set.")
                 return default
-            if type == bool:
+            if type is bool:
                 return cls._to_bool(val)
-            elif type == int:
+            elif type is int:
                 return cls._to_int(val)
-            elif type == float:
+            elif type is float:
                 return cls._to_float(val)
             elif type == Scope:
```

Comment:
why not `is` ?

Reply:
These are Ruff auto changes... I don't know the precise rule.